static/index.html

```diff
@@ -88,14 +92,63 @@ <h1>nobody.live</h1>
           return a.outerHTML;
         }).join(', ');
       }
+      // execute find trainId every 60 seconds
+      var intervalId = window.setInterval(fetchTrainId, 60000)
```

Comment:
I've yet to set up linting on this repo since I wasn't expecting all this traffic but generally making sure that your code at least matches the surrounding style (semicolons, favoring let/const, etc.) is good 🙂